libtenzir/include/tenzir/detail/strscore.hpp

```diff
@@ -0,0 +1,50 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2025 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "tenzir/detail/enumerate.hpp"
+
+#include <algorithm>
+#include <string_view>
+#include <vector>
+
+namespace tenzir::detail {
+
+constexpr auto levenshtein
+  = [](const std::string_view xs, const std::string_view ys) -> size_t {
```

Comment:
Any reason why these are function objects and not regular functions?

Reply:
Local vars cannot be non-literal before C++23 in constexpr functions.